src/components/ConversionResult/ConversionResult.jsx

```diff
@@ -0,0 +1,33 @@
+import "./ConversionResult.scss";
+import Button from "../Button/Button";
+
+/**
+ * Props for Conversion Result (Scroll down in the rollover to see all).
+ * @amount Amount of money in fromCurrency.
+ * @fromCurrency The name of the currency being converted from.
+ * @fromCode The abbreviation for the currency being converted from.
+ * @toCurrency The name of the currency being converted to.
+ * @toCode The abreviation for the currency being converted to.
+ * @rate The covnersion rate between fromCurrency and toCurrency (Not the other way around).
+ * @date The Date element from the Exchage Rates return for fromCurrency.
+ * @handleClick The function to be run when the "Make Transfer" button is clicked.
+ */
```

Comment:
nice addition, but not necessary, fine to keep in but you don't need to do for your next ticket